app/models/reports/schema_v2.rb

```diff
@@ -137,6 +138,23 @@ def missed_visits_with_ltfu_rates
       missed_visits_rates(with_ltfu: true)
     end
 
+    memoize def visited_without_bp_taken(with_ltfu: false)
+      if with_ltfu
+        regions.each_with_object({}) { |region, hsh| hsh[region.slug] = sum(region, :adjusted_visited_no_bp_lost_to_follow_up) }
```

Comment:
Hey, I think the definition here and other ltfu/non-ltfu queries may be incorrect. The `_lost_to_follow_up` numbers in the facility states table has the total for **only** lost to follow up patients, not ltfu+under_care, which is probably what we want here. Do our tests cover this well?

So we should be either summing up the under_care and ltfu numbers here, or exposing ltfu+under_care numbers from the table. In previous discussions about this, I suggested exposing ltfu, and under_care numbers separately so that we can use them as building blocks here in ruby land. Exposing ltfu+under care numbers from the DB would be quite rigid in terms of design.

Reply:
Also, `missed_visits` [defined above](https://github.com/simpledotorg/simple-server/pull/2839/files#diff-e95110f3347ec2a8767b4146482eff9bf0685c7ae55302c630070cbe3411084bR116) does `field = with_ltfu ?`.. it would be good to be consistent in structuring these methods.

Reply:
I actually had the summing code here and then removed it after working thru tests and reconsidering if we needed it.  Now with fresh eyes I think you are right that we need it.  If we do need it, I think we should add it to the facility_states matview to keep the app side code straightforward...the additional logic to sum both fields got a little ugly and also made everything just inconsistent enough in SchemaV2 to be frustrating.

I'm going to reasses this and will open a PR to add the summed columns if need be.

Reply:
@ssrihari Okay, I handled this in code now and updated all the specs. This also gets the numbers much more in line when viewing things in reports, so I think this is good to go after an approval.

We can add it to the matviews next week if we want, though the code to add it was not too bad after all.